src/ui/menu_nbgl.c

```diff
@@ -128,7 +118,7 @@ static void controls_callback(int token, uint8_t index) {
         // to activate the dummy 2 setting
         if (!N_storage.dummy2_allowed) {
             // Display the warning message and ask the user to confirm
-            nbgl_useCaseChoice(&C_warning64px,
+            nbgl_useCaseChoice(&C_Warning_64px,
```

Comment:
I am wondering if we should not provide a `nbgl_useCaseWarning` api. This will provide abstraction to the glyph name, from the app point of view :)